Makefile

```diff
@@ -21,7 +21,7 @@ test: start
 host-build:
 	./checks "./*.go ./ofctrl/ ./ovsdbDriver/ ./pqueue/ ./rpcHub/"
 	go get github.com/tools/godep
-	godep go install ./ ./ofctrl
+	godep go install -race ./ ./ofctrl
```

Comment:
'-race` flag probably needs to be enabled in netplugin not ofnet..
we should probably enable this for unit tests if possible..